dashboard/package.json

```diff
@@ -37,6 +37,17 @@
   "dependencies": {
     "@10xjs/date-input-controller": "0.1.6",
     "@10xjs/form": "^0.1.7",
+    "@babel/core": "^7.0.0",
+    "@babel/plugin-proposal-class-properties": "^7.0.0",
+    "@babel/plugin-proposal-object-rest-spread": "^7.0.0",
+    "@babel/plugin-syntax-dynamic-import": "^7.0.0",
+    "@babel/plugin-transform-destructuring": "^7.0.0",
+    "@babel/plugin-transform-react-jsx": "^7.0.0",
+    "@babel/plugin-transform-react-jsx-self": "^7.0.0",
+    "@babel/plugin-transform-react-jsx-source": "^7.0.0",
```

Comment:
I believe `-self` and `-source` can be removed but I didn't want to until I was sure that the error modal was not using it(?)

Reply:
These are now implicitly added by https://babeljs.io/docs/en/babel-preset-react and aren't necessary here